docs/index.rst

```diff
@@ -2142,30 +2142,27 @@ OpenSALT allows for the associations defined by \ |LINK19|\ . The permitted asso
-+----------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+    +----------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+----------------------------------------------------------------------------------+
+    |Association Type|Description                                                                                                                                                                                                                     |Commentary (Unofficial - not representative of official CASE documentation)       |
+    +================+================================================================================================================================================================================================================================+==================================================================================+
+    |exactMatchOf    |Equivalent to. Used to connect derived CFItem to CFItem in original source CFDocument.                                                                                                                                          |Used to connect derivative framework statements.                                  |
+    +----------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+----------------------------------------------------------------------------------+
+    |exemplar        |The target/destination node is an example of best practice for the definition of the source/origin.                                                                                                                             |Point to a learning resource or other public artifacts(Unlimited).                |
+    +----------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+----------------------------------------------------------------------------------+
+    |hasSkillLevel   |The destination of this association is understood to define a given skill level i.e. Reading Lexile 100, Depth Knowledge 2, or Cognitive Level (Blooms Taxonomy) etc.                                                           |Used to capture equivalencies between academic and Workforce competencies.        |
```

Comment:
I'm not sure this is quite correct.  Reading lexiles, DOK, etc. are not really workforce competencies but I think are more defining a "skill level" that is needed to perform the item.

Reply:
I feel that's the one definition that includes relevant examples, true. 
"Used to capture equivalencies between different types of frameworks." ?

Reply:
That definition of hasSkillLevel, while correct, is too abstract to be helpful. I do like the examples. It is interesting, for the CASE best practices, that hasSkillLevel is similar to rubricCrtieriaLevel which is also a skill level. I am fine with the change with the suggestions.

Reply:
Shall I merge?

Reply:
Please!